data/djangocon-2014/videos/lightning-talks.json

```diff
@@ -1,23 +1,25 @@
 {
-  "category": "DjangoCon 2014",
-  "slug": "lightning-talks",
-  "title": "Lightning Talks",
-  "summary": "By, Various Presenters\n\nHelp us caption & translate this video!\n\nhttp://amara.org/v/FQm0/",
-  "description": "",
-  "quality_notes": "",
-  "language": "English",
-  "copyright_text": "",
-  "thumbnail_url": "https://i.ytimg.com/vi/NMwuzJP4Z_o/maxresdefault.jpg",
-  "duration": 3601,
+  "category": "DjangoCon 2014", 
+  "slug": "lightning-talks", 
+  "title": "Lightning Talks", 
+  "summary": "Help us caption & translate this video!\n\nhttp://amara.org/v/FQm0/", 
+  "description": "", 
+  "quality_notes": "", 
+  "language": "English", 
+  "copyright_text": "", 
+  "thumbnail_url": "https://i.ytimg.com/vi/NMwuzJP4Z_o/maxresdefault.jpg", 
+  "duration": 3601, 
   "videos": [
     {
-      "type": "youtube",
-      "length": 0,
-      "url": "https://www.youtube.com/watch?v=NMwuzJP4Z_o"
+      "url": "https://www.youtube.com/watch?v=NMwuzJP4Z_o", 
+      "length": 0, 
+      "type": "youtube"
     }
-  ],
-  "source_url": "https://www.youtube.com/watch?v=NMwuzJP4Z_o",
-  "tags": [],
-  "speakers": [],
+  ], 
+  "source_url": "https://www.youtube.com/watch?v=NMwuzJP4Z_o", 
+  "tags": [], 
+  "speakers": [
+    "Various Presenters"
```

Comment:
I'd nix this--it's not a real person. For lightning talks, I typically figure out who all the speakers are and then list them individually. We should continue doing that, but we can write up an issue and then do it in a future PR.